doc/user/content/manage/access-control/_index.md

```diff
@@ -10,116 +10,298 @@ menu:
     weight: 15
 ---
 
-This page introduces role-based access management (RBAC) in Materialize. RBAC
-allows you to apply granular privileges to your Materialize objects and clusters. Organizations
-using RBAC can manage user roles and privileges to ensure there is not
-unauthorized or improper access to sensitive objects.
+{{< note >}}
+Configuring and managing access control in Materialize requires **administrator** privileges.
```

Comment:
Creating, altering, dropping, and granting roles can all be done with the `CREATEROLE` system privilege.

Granting/revoking privileges can be done by the role that owns the object who's privileges are being granted/revoked.

Anyone can alter default privileges s.t. the target role is their own role.

Reply:
The purpose of this page is to talk about access control as a whole, not just RBAC. To e.g., invite users or configure SSO, you need to be an `Organization Admin`. It can be confusing to use "administrator privileges" to refer to organization-level privileges, but I couldn't find a better way to make a distinction here.